### PR TITLE
Fixed issues causing citrine builds to stall

### PR DESCRIPTION
**Pull request by sumeetchhetri**
Speed up build times
Removed nginx/apache/redis/memcached tests which were no longer required

### CODE REVIEW

**Comment by sumeetchhetri** [comment]
Have setup tfb tests on my repo to validate the build from now on, [here](https://travis-ci.org/github/sumeetchhetri/ffead-cpp/builds/705101179) is the link for the latest build success

**Comment by sumeetchhetri** [comment]
@nbrady-techempower, can this be merged before the next citrine build please?

File: frameworks/C++/ffead-cpp/install_ffead-cpp-backends.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+cd $IROOT
+
+wget -q https://github.com/sumeetchhetri/ffead-cpp/archive/master.zip
```
Comment (nbrady-techempower) [inline frameworks/C++/ffead-cpp/install_ffead-cpp-backends.sh:5]: This and other places you pull from github must point to a specific release or hash, they can't be a branch or master. Your framework and dependencies can't be updated automatically without a pull request to make sure they pass travis.

Comment (sumeetchhetri) [reply frameworks/C++/ffead-cpp/install_ffead-cpp-backends.sh]: I have pinned the dependencies and the framework source code to specific commit id's now.

**Pull request merged**